examples/todomvc/app.re

```diff
@@ -36,7 +38,10 @@ module Top = {
       None
     };
     let handleChange {state} event =>
-      Some {...state, newTodo: ReasonJs.Document.value event##target};
+      switch (event##target |> Element.asHtmlElement) {
```

Comment:
If this is the case then we might need to revisit the stance on our bindings. Introducing monadic sugar + functors to beginners is a no-go